core/src/test/scala/org/apache/spark/scheduler/OutputCommitCoordinatorSuite.scala

```diff
@@ -289,6 +289,7 @@ private case class OutputCommitFunctions(tempDirPath: String) {
   // Mock output committer that simulates a failed commit (after commit is authorized)
   private def failingOutputCommitter = new FakeOutputCommitter {
     override def commitTask(taskAttemptContext: TaskAttemptContext): Unit = {
+      super.commitTask(taskAttemptContext)
```

Comment:
I'm a bit worried about changing the behavior for the existing testing output committer. Can we create a new one for the new test case?

Reply:
Revert this change, not need this change.